src/inmanta/protocol/decorators.py

```diff
@@ -85,7 +85,10 @@ def method(
         :param server_agent: This is a call from the Server to the Agent (reverse http channel through long poll)
         :param agent_server: This is a call from the Agent to the Server
         :param validate_sid: This call requires a valid session, true by default if agent_server and not api
-        :param client_types: The allowed client types for this call
+        :param client_types: The allowed client types for this call. The valid values are:
```

Comment:
I would keep the documentation in one place on link to it. Otherwise we have to update the documentation in all places if we ever add an extra client_type. I would put the documentation next to the VALID_CLIENT_TYPES list.